dev/eo/helmfile.yaml

```diff
@@ -0,0 +1,48 @@
+# single env example file
+# helm file dependencies
+# helm plugin install https://github.com/databus23/helm-diff
+# repositories to be installed
+repositories:
+  - name: datarepo-helm
+    url: https://broadinstitute.github.io/datarepo-helm
+
+# helm releases to be deployed
+releases:
+  - name: eo-jade-create-secret-manager-secret    # release name
+    namespace: eo   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/create-secret-manager-secret   # chart name
+    version: 0.0.6   #    Chart version
```

Comment:
Ah!  Locally throughout my `ok` helmfile, I ultimately removed all `version` specifications for each of these releases.  The effect of that removal is that the latest version of these apps will be used any time I redeploy.  Otherwise, you'll want to update the versions to match the latest ones: https://github.com/broadinstitute/datarepo-helm/blob/master/charts/datarepo/Chart.lock

Here is a bit more context: https://github.com/broadinstitute/datarepo-helm-definitions/pull/444#discussion_r844039403

Reply:
(For posterity, these versions that you had initially *were* the latest ones, and I visually mismapped them :) )